core-helm-chart/templates/operator.yaml

```diff
@@ -203,8 +203,8 @@ spec:
         - --wds-name={{.Values.ControlPlaneName}}
         - --api-groups={{.Values.APIGroups}}
         - -v=2
-        image: ghcr.io/kubestellar/kubestellar/kubestellar-operator:0.20.0-alpha.1
-        imagePullPolicy: IfNotPresent
+        image: {{ .Values.image.repository }}:{{ .Values.image.tag }}
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
```

Comment:
Note that these values are overwritten by `make chart`

Reply:
Good point.